doc/source/conf.py

```diff
@@ -217,4 +226,6 @@ def sort_authors(filename):
 # rst-epilog implements substitutions
 rst_epilog = """
 .. |MDAnalysis_version| replace:: {0}
-""".format(MDAnalysis_version)
+""".format(
+    MDAnalysis_version
+)
```

Comment:
This is one of the weird ones, but harmless. And would be a non-issue with f-strings.

Maybe a first good step to apply black to the main code base would be to convert `format` statements into f-strings.

Reply:
Good shout, I turned this into an f-string